spec/requests/swagger_spec.rb

```diff
@@ -3270,6 +3270,10 @@
     end
 
     describe 'contact us' do
+      before do
```

Comment:
These were failing because some expectations were validating that some toggle was set and the new Flipper call in saved_claim was expecting it. I added this to default all to call the original so any that don't have explicit expectations won't fail.